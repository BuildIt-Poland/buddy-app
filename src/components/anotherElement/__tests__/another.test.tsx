import React from 'react';
import { create } from 'react-test-renderer';

import Another from '../another';

describe('Component - another', () => {
  test('renders correctly', () => {
    const component = create(<Another />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
