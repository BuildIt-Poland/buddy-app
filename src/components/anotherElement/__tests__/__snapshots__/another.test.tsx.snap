// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - another renders correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body1"
>
  Buddy teste
</p>
`;
